luxos.py

```diff
@@ -512,6 +529,14 @@ def execute_command(host: str, port: int, timeout_sec: int, cmd: str,
             for thread in threads:
                 thread.join()
 
+            # Introduce the batch delay if specified
```

Comment:
Idem before, instead of waiting between batches, we should be waiting between N elements processed. That way we are effectively delaying every N elements and removing the individual miner delay from the loop.

Reply:
As reviewed with Guz, we will come back to this helper file later, so approving as delay addition is ok.